tests/components/Innsending.4.Kvittering.test.tsx

```diff
@@ -88,7 +88,7 @@ describe("Kvittering", () => {
 
     const button = await waitFor(() => screen.findByText("overskrift.etterregistrertMeldekort"));
     button.click();
-    expect(replaceMock).toBeCalledWith("undefined/etterregistrer-meldekort/2"); // BASE_PATH/etterregistrer-meldekort/w
+    expect(replaceMock).toBeCalledWith("/etterregistrer-meldekort/2");
```

Comment:
github actions / test klager på `AssertionError: expected "spy" to be called with arguments: [ '/etterregistrer-meldekort/2' ]`. Skal det løses?

Reply:
Fikset